config/webpack.lib.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = env => ({
     filename: "index.js",
     sourceMapFilename: "[file].map",
     library: "storyhooks",
-    libraryTarget: "umd"
+    libraryTarget: "umd",
+    globalObject: "typeof self !== 'undefined' ? self : this"
```

Comment:
Could you please elaborate on what this actually does? :)

Reply:
Sorry! Yes, this comes from https://github.com/webpack/webpack/issues/6522 and tells webpack to use the specified string as the global object in it's UMD module definition. Something broke in the way webpack@4 outputs that UMD header and this line is required to make your lib work with SSR (it has no impact on browser instantiation).

Unfortunately, like I said in the description, the react-json-view dependency will also have to do this upstream in order for retoggle to work in SSR.

Reply:
Thank you. I'll merge this for now.